lightspeed/modules/proc_install-aap-lightspeed-operator.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: PROCEDURE
+
+[id="install-aap-lightspeed-operator_{context}"]
+
+= Installing the {PlatformName} operator
+
+Use this procedure to install the {PlatformNameShort} operator on the {OCP}. 
+
+.Prerequisites
+* You have installed and configured automation controller.
```

Comment:
Use the automation controller attribute here-- {ControllerName}

Reply:
Done